pipelines/build/Jenkinsfile

```diff
@@ -119,67 +119,92 @@ node('splice-standalone') {
                     hadoop_version = "2.6.0-\\\${envClassifier.value}"
                     kafka_version = "0.10.0-kafka-2.1.0"
+                    spark_version = "2.2.0.cloudera2"
                 } else if ("$env_classifier" == "cdh5.14.0"){
                     spark_folder = "."
                     hbase_version = "1.2.0-\\\${envClassifier.value}"
                     hadoop_version = "2.6.0-\\\${envClassifier.value}"
                     kafka_version = "0.10.0-kafka-2.1.0"
+                    spark_version = "2.2.0.cloudera2"
                 } else if ("$env_classifier" == "cdh5.14.2"){
                     spark_folder = "."
                     hbase_version = "1.2.0-\\\${envClassifier.value}"
                     hadoop_version = "2.6.0-\\\${envClassifier.value}"
                     kafka_version = "0.10.0-kafka-2.1.0"
+                    spark_version = "2.2.0.cloudera2"
                 } else if ("$env_classifier" == "cdh5.16.1"){
                     spark_folder = "."
                     hbase_version = "1.2.0-\\\${envClassifier.value}"
                     hadoop_version = "2.6.0-\\\${envClassifier.value}"
                     kafka_version = "0.10.0-kafka-2.1.0"
+                    spark_version = "2.2.0.cloudera2"
                 } else if ("$env_classifier" == "cdh6.3.0"){
                     spark_folder = "spark2.4"
                     hbase_version = "2.1.0-\\\${envClassifier.value}"
                     hadoop_version = "3.0.0-\\\${envClassifier.value}"
                     kafka_version = "2.2.1-\\\${envClassifier.value}"
+                    spark_version = "2.4.0-\\\${envClassifier.value}"
                 } else if ("$env_classifier" == "hdp2.6.1"){
                     spark_folder = "."
                     hbase_version = "1.1.2.2.6.1.0-129"
                     hadoop_version = "2.7.3.2.6.1.0-129"
                     kafka_version = "0.10.1.2.6.1.0-129"
+                    spark_version = "2.1.1.2.6.1.0-129"
                 } else if ("$env_classifier" == "hdp2.6.3"){
                     spark_folder = "."
                     hbase_version = "1.1.2.2.6.3.0-235"
                     hadoop_version = "2.7.3.2.6.3.0-235"
                     kafka_version = "0.10.1.2.6.3.0-235"
+                    spark_version = "2.2.0.2.6.3.0-235"
                 } else if ("$env_classifier" == "hdp2.6.4"){
                     spark_folder = "."
                     hbase_version = "1.1.2.2.6.4.0-91"
                     hadoop_version = "2.7.3.2.6.4.0-91"
                     kafka_version = "0.10.1.2.6.4.0-91"
+                    spark_version = "2.2.0.2.6.4.0-91"
                 } else if ("$env_classifier" == "hdp2.6.5"){
                     spark_folder = "spark2.3"
                     hbase_version = "1.1.2.2.6.5.0-292"
                     hadoop_version = "2.7.3.2.6.5.0-292"
                     kafka_version = "1.0.0.2.6.5.0-292"
+                    spark_version = "2.3.0.2.6.5.0-292"
```

Comment:
Do we still need to support hdp2.6.x?

Reply:
I kept the logic inside, but it'll be inactive since I removed it from the available platforms to select from. I'll keep the logic in the pipeline until everyone migrates to branch-3.x